src/components/SquarePaymentForm.test.tsx

```diff
@@ -1,12 +1,12 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import * as React from 'react'
 import { shallow } from 'enzyme'
 import { expect } from 'chai'
 
 import SquarePaymentForm from './SquarePaymentForm'
 
 describe('SquarePaymentForm', () => {
-
-  let wrapper: any // eslint-disable-line @typescript-eslint/no-explicit-any
+  let wrapper: any
```

Comment:
Do we want `any`, or could we use `unknown`?

Reply:
We get a lot of "Property 'X' does not exist on type 'unknown'". I figured this was just for a test so it was fine for now.